components/event-sources/reconciler/httpsource/reconcile.go

```diff
@@ -233,6 +233,7 @@ func (r *Reconciler) makeKnService(src *sourcesv1alpha1.HTTPSource,
 		object.WithProbe(adapterHealthEndpoint),
 		object.WithControllerRef(src.ToOwner()),
 		object.WithLabel(routeconfig.VisibilityLabelKey, routeconfig.VisibilityClusterLocal),
+		object.WithSpecTemplateLabel(eventMeshLabel, "true"),
```

Comment:
An empty struct would be enough and signal that we only care about the label being _present_ (it would make no sense to set `event-mesh: "false"` right?)